src/help.coffee

```diff
@@ -77,16 +77,17 @@ module.exports = (robot) ->
     else
       msg.reply emit
 
-  robot.router.get "/#{robot.name}/help", (req, res) ->
-    cmds = renamedHelpCommands(robot).map (cmd) ->
-      cmd.replace(/&/g,'&amp;').replace(/</g,'&lt;').replace(/>/g,'&gt;')
+  if !process.env.HUBOT_HELP_DISABLE_HTTP?
```

Comment:
I reckon you could just return immediately if `HUBOT_HELP_DISABLE_HTTP` is `true`. Smaller diff too.